docs/sources/flow/concepts/modules.md

```diff
@@ -0,0 +1,145 @@
+---
+aliases:
+- ../../concepts/modules/
+title: Modules
+weight: 300
+---
+
+# Modules
+
+_Modules_ are a way to create Grafana Agent Flow configuration files which can
+laoded as a component. Modules are a great way to parameterize configuration
+files to create reusable pipelines.
+
+Modules are Grafana Agent Flow configuration files which have:
```

Comment:
We should perhaps avoid calling them files if they are not necessarily files. Is there a better word here?

Reply:
Maybe just "configurations?"